ui/app/components/auth/tabs.hbs

```diff
@@ -0,0 +1,50 @@
+{{!
+  Copyright (c) HashiCorp, Inc.
+  SPDX-License-Identifier: BUSL-1.1
+~}}
+
+<Hds::Tabs @onClickTab={{@onTabClick}} @selectedTabIndex={{@selectedTabIndex}} as |T|>
+  {{#each-in @authTabs as |methodType mounts|}}
+    <T.Tab data-test-auth-method={{methodType}}>{{@displayNameHelper methodType}}</T.Tab>
+    <T.Panel>
+      <div class="has-top-padding-m">
+        {{! elements "behind" tabs always render on the DOM and are just superficially hidden/shown 
+        for accessibility we only want to render form inputs relevant to the selected method 
+        wrapping the elements in this conditional only renders them when the tab is selected }}
```

Comment:
kind of a run...on.... let me know if this could be reworded to make more sense! 😅

Reply:
Yep! I can add punctuation, thanks!